packages/bsky/src/auth-verifier.ts

```diff
@@ -182,6 +205,51 @@ export class AuthVerifier {
     }
   }
 
+  entrywaySession = async (reqCtx: ReqCtx): Promise<StandardOutput> => {
+    const token = bearerTokenFromReq(reqCtx.req)
+    if (!token) {
+      throw new AuthRequiredError(undefined, 'AuthMissing')
+    }
+
+    // if entryway jwt key not configured then do not parsed these tokens
+    if (!this.entrywayJwtPublicKey) {
+      throw new AuthRequiredError('Malformed token', 'InvalidToken')
+    }
+
+    const res = await jose
+      .jwtVerify(token, this.entrywayJwtPublicKey)
+      .catch((err) => {
+        if (err?.['code'] === 'ERR_JWT_EXPIRED') {
+          throw new AuthRequiredError('Token has expired', 'ExpiredToken')
+        }
+        throw new AuthRequiredError(
+          'Token could not be verified',
+          'InvalidToken',
+        )
+      })
+
+    const { sub, aud, scope } = res.payload
+    if (typeof sub !== 'string' || !sub.startsWith('did:')) {
+      throw new AuthRequiredError('Malformed token', 'InvalidToken')
+    } else if (
+      typeof aud !== 'string' ||
+      !aud.startsWith('did:web') ||
```

Comment:
nit (?) the method name should probably not be alterable

```suggestion
      !aud.startsWith('did:web:') ||
```